### DISCUSSION TITLE
I need help with adding the amount of clicks on another site

### DISCUSSION THREAD

**Discussion by alexwalker0** [YOURLS/YOURLS#2879] [Q&A] [answered]
I want to add the number of click on another website, so I can display a live / real-time click amount. Could anyone please help me with that?

Thank you,
Alex

**Comment by ozh** [comment, marked as answer]
The YOURLS API is here for you. On your site you simply need to fetch data from http://sho.rt/yourls-api.php?action=stats

See the readme about the API, the sample file in the root as an example, and maybe https://github.com/YOURLS/YOURLS/wiki/PasswordlessAPI

**Reply by alexwalker0** [reply]
Hi,

Thank you for your reply.

The link "http://sho.rt/yourls-api.php?action=stats" doesn't work

Thank you, 
Alex

**Reply by LeoColomb** [reply]
@alexwalker0 This link is an example, you'll need to replace sho.rt with your own domain.

**Reply by alexwalker0** [reply]
Thank you so much
